modules/FindBasisUniversal.cmake

```diff
@@ -174,24 +174,19 @@ foreach(_component ${BasisUniversal_FIND_COMPONENTS})
                     ${BasisUniversalEncoder_DIR}/basisu_resampler.cpp
                     ${BasisUniversalEncoder_DIR}/basisu_resample_filters.cpp
                     ${BasisUniversalEncoder_DIR}/basisu_ssim.cpp
-                    ${BasisUniversalEncoder_DIR}/basisu_uastc_enc.cpp
-                    ${BasisUniversalEncoder_DIR}/jpgd.cpp)
+                    ${BasisUniversalEncoder_DIR}/basisu_uastc_enc.cpp)
 
                 # Files not present in all supported basis versions, treat them
                 # as optional and do nothing if not found.
                 foreach(_file
                     # Removed in 1.16
-                    apg_bmp.c
                     basisu_astc_decomp.cpp
                     basisu_global_selector_palette_helpers.cpp
-                    lodepng.cpp
                     # Added in 1.16
                     basisu_opencl.cpp
-                    pvpngreader.cpp
                     # Added in 1.50
                     basisu_astc_hdr_enc.cpp
-                    3rdparty/android_astc_decomp.cpp
-                    3rdparty/tinyexr.cpp)
+                    3rdparty/android_astc_decomp.cpp)
```

Comment:
I was checking if this bit could be removed / stubbed as well, but I'm not sure. To me it doesn't make sense why an _encoder_ would need to decode ASTC blocks if the input is always RGBA, apart from some debug-only opt-in validation.

Reply:
Yeah, from a quick glance it's used with the `validate` option(s), so can't be removed unfortunately.

Reply:
It's not _massive_ at least :) 

Actually I wasn't aware there was such a self-contained ASTC decoding thingy, I may integrate that as another plugin next to bcdec and friends. In comparison, https://github.com/richgel999/astc_dec doesn't seem to have a HDR variant yet.